### PR TITLE
chore(project name): PNRN-30 change project name

### PR DESCRIPTION
**Pull request by princhcanal**
## Description

- change project name in package.json

## Checklist

_First, create a draft pull request. Then mark the PR as ready for review when all necessary checkbox items has been completed_

- [x] Reviewed own code
- [x] Commented on code that is hard to understand
- [ ] Implemented tests for the feature/bugfix
- [x] All GitHub status checks pass
- [x] The frontend and backend PR previews have been deployed

**Backend only**

- [ ] Generated the client api if there are new or deleted endpoints and/or DTOs

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "frontend",
+  "name": "teknoy-ems-frontend",
```
Comment (princhcanal) [inline frontend/package.json:2]: this is changing the name on the frontend

**Comment by beefysalad** [comment]
LGTM

**Review by beefysalad — APPROVED** [review]
LGTM

**Review by noice-noise — DISMISSED** [review]
'Tis good ser.

**Review by beefysalad — CHANGES_REQUESTED** [review]
hmmmm i think you can do better

**Review by noice-noise — COMMENTED** [review]
Hmm, ayus!

**Review by beefysalad — APPROVED** [review]

**Pull request merged**